src/main/resources/init_db.sql

```diff
@@ -1,4 +1,43 @@
-CREATE DATABASE `internet_shop` DEFAULT CHARACTER SET utf8;
+
+
+DROP TABLE IF EXISTS `order_products`;
```

Comment:
let's remove all the DROP statements from here
they should be executed in particular order to be applied

Reply:
Sad, ok removed.